tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -184,3 +192,171 @@ setup_file() {
     run bash -c 'fvm self uninstall --yes'
     assert_success
 }
+
+@test "Copies binaries to Fluvio Binaries Directory when using fvm switch" {
+    run bash -c '$FVM_BIN self install'
+    assert_success
+
+    # Sets `fvm` in the PATH using the "env" file included in the installation
+    source ~/.fvm/env
+
+    # Removes Fluvio Directory
+    rm -rf $FLUVIO_HOME_DIR
+
+    # Ensure `~/.fluvio` is not present
+    run bash -c '! test -d $FLUVIO_HOME_DIR'
+    assert_success
+
+    # Installs Stable Fluvio
+    run bash -c 'fvm install stable'
+    assert_success
+
+    # Switch version to use Stable Fluvio
+    run bash -c 'fvm switch stable'
```

Comment:
Duplicate user's flow.  Install `stable` and `0.10.15` or some version then switch